src/interactive/results.ts

```diff
@@ -182,19 +182,24 @@ interface Frame extends _Frame {
     err: string | vscode.MarkdownString,
 }
 
-let stackFrame: Frame[] = []
+let stackFrame: Result[] = []
 export function setStackTrace(err: string, stackframe: Array<_Frame>) {
-    stackFrame = stackframe.map(frame => {
+    stackFrame.forEach(r => {
+        r.remove()
+    })
+
+    stackFrame = []
+
+    drawStackFrame(stackframe.map(frame => {
         return {
             path: frame.path,
             line: frame.line,
             err
         }
-    })
-    refreshStackTrace()
+    }))
```

Comment:
Actually this change has one regression; results are now only registered for currently open editors, meaning we can't get error highlight for editors that are opened later.
Will make a change on this anyway.

Reply:
Cool, let me know when you want another review on this :)

Reply:
Will do ;)